### PR TITLE
add extract_dict function

### PR DESCRIPTION
**Pull request by HLasse**
Any comments?

### CODE REVIEW

File: textdescriptives/dataframe_extract.py [hunk 1]
```diff
@@ -139,3 +190,4 @@ def extract_df(
     "n_sentences",
     "n_characters",
 ]
+
```
Comment (KennethEnevoldsen) [inline textdescriptives/dataframe_extract.py:193]:
```suggestion
```

File: textdescriptives/tests/test_extractor.py [hunk 2]
```diff
@@ -43,3 +43,19 @@ def test_extract_df_error(nlp):
         td.extract_df(doc, metrics="not a metric")
     with pytest.raises(Exception) as e_info:
         td.extract_df(doc, metrics=True)
+
```
Comment (KennethEnevoldsen) [inline textdescriptives/tests/test_extractor.py:46]: It looks like test_extract_df_subsetters does not check these properly (i.e. they have no effect):
df[td.readability_cols]
    df[td.dependency_cols]
    df[td.descriptive_stats_cols]
    
apparently you can't correct old stuff.

File: README.md [hunk 3]
```diff
@@ -35,18 +35,19 @@ doc._.readability
 doc._.token_length
 ```
 
-TextDescriptives includes a convenience function for extracting metrics to a Pandas DataFrame
+TextDescriptives includes convenience functions for extracting metrics to a Pandas DataFrame or a dictionary.
 
 ```py
 td.extract_df(doc)
+# td.extract(dict(doc))
```
Comment (KennethEnevoldsen) [inline README.md:42]:
```suggestion
# td.extract_dict(doc)
```

**Review by KennethEnevoldsen — APPROVED** [review]
Looks great!

File: textdescriptives/tests/test_extractor.py (hunk 2 above)
Comment (HLasse) [reply textdescriptives/tests/test_extractor.py:46]: I'll check it out - thanks

**Comment by HLasse** [comment]
fixed the test of subsetters

**Pull request merged**